custom-namespaces.yaml

```diff
@@ -750,4 +750,8 @@ websites:
     blogs_daily_summary:
       type: table_view
       tables:
-        - table: moz-fx-data-marketing-prod.ga_derived.blogs_daily_summary_v1
\ No newline at end of file
+        - table: moz-fx-data-marketing-prod.ga_derived.blogs_daily_summary_v1
+    blogs_landing_page_summary:
+      type: table_view
+      tables:
+        - table: moz-fx-data-marketing-prod.ga_derived.blogs_landing_page_summary_v1
```

Comment:
Curious why `ga_derived` doesn't follow the same view conventions, shouldn't there be a `ga.blogs_landing_page_summary`?